medusa/scene_exceptions.py

```diff
@@ -20,22 +20,71 @@
 from __future__ import unicode_literals
 
 import datetime
+import logging
 import threading
 import time
 
 import adba
 from medusa.indexers.indexer_api import indexerApi
 from six import iteritems, text_type
-from . import app, db, helpers, logger
+from . import app, db, helpers
 from .indexers.indexer_config import INDEXER_TVDBV2
 
-exceptionsCache = {}
-exceptionsSeasonCache = {}
+exceptions_cache = {}
+exceptions_season_cache = {}
 
 exceptionLock = threading.Lock()
 
 xem_session = helpers.make_session()
 
+log = logging.getLogger(__name__)
```

Comment:
I think we usually call this ```logger```.
For consistency reasons I would call it like that instead:
```
logger = logging.getLogger(__name__)
```